dftimewolf/lib/collectors/grr_hunt.py

```diff
@@ -247,20 +259,34 @@ def _extract_hunt_results(self, output_file_path):
     # Extract items from archive by host for processing
     collection_paths = []
     clients = set()
```

Comment:
It looks like this should actually be client_ids, please rename.

Reply:
Done.